usr/src/uts/armv8/os/fakebop.c

```diff
@@ -454,6 +876,15 @@ read_bootenvrc(void)
 			DBG(physmem);
 		}
 	}
+	/* XXXARM: early_allocation = 0; */
+
+#if XXXARM
```

Comment:
Is the comment saying why this can't be live now?  It seems reasonable to have it

Reply:
This is disabled while we get the boot console up. The commented code above is something I need to dig into in early i86pc memory management to see if we need it here.